src/routes/router.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const mainRoute = require ('./main/main');
+const signUpRoute = require ('./users/sign-up-route');
+const getProducts = require ('./products/products');
+const getUsers = require('./users/sign-up-route');
+const getOrder = require('./products/orders');
+const apiRoutes = express.Router();
+
+apiRoutes
+    .get('/', mainRoute)
+    .get('/signup', signUpRoute)
```

Comment:
why /signup don't have slash at the end but products/ have slash?

whats point?